shell/platform/embedder/embedder_external_texture_gl.cc

```diff
@@ -68,6 +70,8 @@ sk_sp<SkImage> EmbedderExternalTextureGL::ResolveTexture(
 
   GrBackendTexture gr_backend_texture(width, height, GrMipMapped::kNo,
                                       gr_texture_info);
+  context->flushAndSubmit();
+  context->resetContext(kAll_GrBackendState);
```

Comment:
The destruction callback is not called here. Instead. It will be called by Skia when the last reference to the texture is removed. The right way to do this would be to wrap the proc in a closure that flushes the context ([this pattern for reference](https://github.com/flutter/engine/blob/6b5f5c34e720f588086f369de8b8ac1743ddee72/shell/platform/embedder/embedder.cc#L439)).

Reply:
Sorry, the link to the reference in the last comment was bad. [This is the correct link](https://github.com/flutter/engine/blob/6b5f5c34e720f588086f369de8b8ac1743ddee72/shell/platform/embedder/embedder.cc#L539).

Reply:
Oh crap - yeah; serves me right for banging this up at the last minute before shutting my machine down for the night. On it.

Reply:
Got back to this today; thanks -- done. Funny, this slipped my mind; I landed a null-safety patch in that exact code the other day.